sormas-ui/src/main/java/de/symeda/sormas/ui/login/DefaultPasswordOtherScreen.java

```diff
@@ -0,0 +1,57 @@
+package de.symeda.sormas.ui.login;
+
+import com.vaadin.ui.*;
+import de.symeda.sormas.api.FacadeProvider;
+import de.symeda.sormas.api.user.UserDto;
+import de.symeda.sormas.ui.utils.ButtonHelper;
+
+import java.util.List;
+
+public class DefaultPasswordOtherScreen extends Window {
+
+    public DefaultPasswordOtherScreen(Runnable onContinue, List<UserDto> otherUsersWithDefaultPassword) {
+        setCaption("Default passwords");
+        setWidth(600.0f, Unit.PIXELS);
```

Comment:
Is hard-coded pixels a requirement of sormas? can we use non absolute vals here?